### ISSUE TITLE
Cannot display the input method popups in wayland applications

### ISSUE THREAD

**Issue by wiwyil2tr** [WillPower3309/swayfx#301]
### Please fill out the following:
- **Description:**
Please describe in plain English what the enhancement is and what the use case is.

OS: Arch Linux
swayfx: 0.3.2
fcitx5: 5.1.8-1  

Cannot display the input method popups in wayland applications under swayfx like kitty, alacritty, but it can be displayed normally in xwayland apps or under sway

swayfx and kitty(can't display popup):
![Screenshot_20240427_110201](https://github.com/WillPower3309/swayfx/assets/108447154/5f610884-141d-4e7c-bdc2-2659c5067264)

swayfx and mate-terminal(normal):
![image](https://github.com/WillPower3309/swayfx/assets/108447154/6d5cbc96-66a9-49a6-9658-0095b9e7f76e)

sway and kitty(normal):
![image](https://github.com/WillPower3309/swayfx/assets/108447154/d61a529b-5680-4e3c-9609-a2a8159f4225)

**Comment by WillPower3309** [comment]
Thanks for raising this issue! Are you able to confirm if this is an issue on upstream sway as well? If so you may want to raise the issue there so that it's fixed for a wider audience. If not, it helps us narrow down where the problem lies :)

**Comment by wiwyil2tr** [comment]
> Thanks for raising this issue! Are you able to confirm if this is an issue on upstream sway as well? If so you may want to raise the issue there so that it's fixed for a wider audience. If not, it helps us narrow down where the problem lies :)

This is an issue that has already been resolved on upstream sway. The input method popup support has been [merged](https://github.com/swaywm/sway/pull/7226) but not yet made into stable releases.

**Comment by mochaaP** [comment]
You can cherry-pick the same changes for 1.9 from AUR `sway-im`.

<https://pagure.io/mochaa-rpms/swayfx/raw/rawhide/f/0001-text_input-Implement-input-method-popups.patch>

**Comment by mochaaP** [comment]
@WillPower3309 I've tested the patch for a few days, and it works fine so far. Are you interested in picking it?
